Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Extensions/ExceptionExtensions.cs

```diff
@@ -1,29 +1,29 @@
-using Amazon.Lambda.TestTool.Models;
```

Comment:
whitespace changes i think are due to phils code not having CRLF line endings. can review the pr with "ignore" whitespace flag setting enabled in meantime